src/ontology/uberon-edit.obo

```diff
@@ -222485,6 +222485,16 @@ relationship: dc-contributor https://orcid.org/0000-0003-2699-3066
 relationship: part_of UBERON:7500094 ! tubercle of calcaneus
 property_value: dcterms-date 2022-05-19T21:16:42Z xsd:dateTime
 
+[Term]
+id: UBERON:7500106
+name: hormion
+def: "Craniometric point along the midline of the sphenovomerine suture on the ventral (inferior in humans) side." [https://locatelambda.org/manual, ISBN:9780873659505]
```

Comment:
Note: [URL in dbxref](https://locatelambda.org/manual) has the following:
defintion: "The most posterior midline point on the vomer."
comment: "Hormion is taken posterior to the vomer at the midline, not into the anterior deflection if it is present."
illustration: http://locatelambda.org/wp-content/uploads/2018/02/hormion.jpg

and from https://medical-dictionary.thefreedictionary.com/hormion:
"A craniometric point at the junction of the posterior border of the vomer with the sphenoid bone." 

Cannot access ISBN.

@megbalk,
Would something like this work?
"A craniometric point at the junction of the posterior midline border of the vomer with the sphenoid bone."
dbxrefs above.